src/js/components/InfiniteScroll/__tests__/InfiniteScroll-test.js

```diff
@@ -75,4 +76,52 @@ describe('InfiniteScroll', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('length when step < array', () => {
```

Comment:
Reading the descriptions/names of the tests, I am able to understand the test scenario, but it is not immediately clear what the expected results or outcome should be.

Could these be written in the form of "Should [expectation / outcome] when [scenario]?" For example, "Should render items equal to the length of step when step < items.length" or something similar. I can reason with the code to try to interpret the intent, but naming in this form might be helpful. Thoughts?

Reply:
I agree, I think the `should ---> when` format provides a good amount of description.

Reply:
I renamed the tests to variations of `Should render items equal to the length of 
  step + 1 when step < items.length`. I agree it is more descriptive than what the tests were named previously.